src/test/firestore/encodeFirestoreValues.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from "chai";
```

Comment:
This file's source file is called `encodeFirestoreValue.ts` but this one is plural, should change to make consistent.

Reply:
Yeah, weird. I'll fix it.